app/src/main/java/org/openobservatory/ooniprobe/utils/VersionUtils.java

```diff
@@ -0,0 +1,11 @@
+package org.openobservatory.ooniprobe.utils;
+
+import org.openobservatory.ooniprobe.BuildConfig;
+
+public class VersionUtils {
+
+    public static String get_software_version(){
+        String release_name = "-rc.5";
```

Comment:
Can we put this in a place where we have defines?

Reply:
We have the defines in model.OONITests.

Reply:
Cool, then put the global version information in there.

Reply:
Or maybe put it in the manifest like this: https://stackoverflow.com/a/7929059

Reply:
Even better, I moved it in the build.gradle file